### PR TITLE
Correct summary fragments in javadocs

### PR DESCRIPTION
**Pull request by andrey-lavrov**

### CODE REVIEW

File: client/src/main/java/org/spine3/base/Events.java [hunk 1]
```diff
@@ -82,25 +78,19 @@ public static void sort(List<Event> events) {
...
+    /** Creates a new {@code Event} instance with the passed event message and context. */
     @SuppressWarnings("OverloadedMethodsWithSameNumberOfParameters")
     public static Event createEvent(Message event, EventContext context) {
         return createEvent(toAny(event), context);
     }
 
-    /**
-     * Creates a new {@code Event} instance with the passed event message wrapped to {@link Any} and context.
-     */
+    /** Creates a new {@code Event} instance with the passed event message wrapped to {@link Any} and context. */
```
Comment (AlexanderLitus) [inline client/src/main/java/org/spine3/base/Events.java]: I guess it is too long for the summary. Pls leave only `Creates a new {@code Event} instance` on the first line. And do the same where possible.

File: server/src/main/java/org/spine3/server/command/CommandHandler.java [hunk 2]
```diff
@@ -45,23 +45,23 @@
 /**
  * The abstract base for classes that expose command handling methods
  * and post their results to {@link EventBus}.
- *
+ * <p/>
```
Comment (AlexanderLitus) [inline server/src/main/java/org/spine3/server/command/CommandHandler.java]: Fail. pls remove such tags.

File: server/src/test/java/org/spine3/server/clientservice/Given.java [hunk 3]
```diff
@@ -111,23 +109,17 @@ private Command() {
             return result;
         }
 
-        /**
-         * Creates a new {@link org.spine3.base.Command} with default properties (current time etc).
-         */
+        /** Creates a new {@link org.spine3.base.Command} with default properties (current time etc). */
```
Comment (AlexanderLitus) [inline server/src/test/java/org/spine3/server/clientservice/Given.java]: pls replace this FQN with just `command`.

**Comment by AlexanderLitus** [comment]
Pls also scan all docs which are longer than one line. It is needed to fix their summaries, too.

**Comment by alexander-yevsyukov** [comment]
When do we finish with this PR?

**Comment by AlexanderLitus** [comment]
When @andrey-lavrov finishes with BKO, he will finish this PR.

**Comment by AlexanderLitus** [comment]
LGTM.

**Pull request merged**